Defs/Drugs/Psychite_Psycho.xml

```diff
@@ -0,0 +1,309 @@
+              <offset>0.33</offset>
+            </li>
+          </capMods>
+          <statFactors>
+            <Suppressability>0</Suppressability>
+          </statFactors>
+        </li>
+        <li>
+          <minSeverity>0.9375</minSeverity>
+          <label>kicking in</label>
+          <painFactor>0.5</painFactor>
+          <restFallFactor>0.5</restFallFactor>
+          <capMods>
+            <li>
+              <capacity>Consciousness</capacity>
+              <offset>0.1</offset>
+            </li>
+            <li>
+              <capacity>Sight</capacity>
+              <offset>0.1</offset>
+            </li>
+            <li>
+              <capacity>Moving</capacity>
+              <offset>0.2</offset>
+            </li>
+          </capMods>
+          <statFactors>
+            <Suppressability>0.5</Suppressability>
+          </statFactors>
+        </li>
+      </stages>
+  </HediffDef>
+
+  <ThoughtDef>
+    <defName>PsychoHigh</defName>
+    <workerClass>ThoughtWorker_Hediff</workerClass>
+    <hediff>PsychoHigh</hediff>
+    <validWhileDespawned>true</validWhileDespawned>
+    <stages>
+      <li>
+        <label>coming down from psycho</label>
+        <description>Something is wrong.</description>
+        <baseMoodEffect>0</baseMoodEffect>
+      </li>
+      <li>
+        <label>barely high on psycho</label>
+        <description>I still feel so good.</description>
+        <baseMoodEffect>20</baseMoodEffect>
+      </li>
+      <li>
+        <label>high on psycho</label>
+        <description>I AM GOD!</description>
+        <baseMoodEffect>50</baseMoodEffect>
+      </li>
+      <li>
+        <label>almost high on psycho</label>
+        <description>I feel like a god.</description>
+        <baseMoodEffect>40</baseMoodEffect>
+      </li>
+    </stages>
+  </ThoughtDef>
+  
+  <HediffDef>
+    <defName>PsychoCrash</defName>
+    <label>psycho crash</label>
+    <labelNoun>a psycho crash</labelNoun>
+    <description>A very unpleasant delayed after-effect of psycho intake.</description>
+    <hediffClass>HediffWithComps</hediffClass>
+    <initialSeverity>1</initialSeverity>
+    <scenarioCanAdd>false</scenarioCanAdd>
+    <comps>
+      <li Class="HediffCompProperties_SeverityPerDay">
+        <severityPerDay>-0.5</severityPerDay>
+      </li>
+    </comps>
+    <stages>
+      <li>
+        <label>recovering</label>
+          <capMods>
+            <li>
+              <capacity>Consciousness</capacity>
+              <offset>-0.2</offset>
+            </li>
+          </capMods>
+      </li>
+      <li>
+        <minSeverity>0.2</minSeverity>
+        <label>serious</label>
+		<vomitMtbDays>0.5</vomitMtbDays>
+          <capMods>
+            <li>
+              <capacity>Consciousness</capacity>
+              <offset>-0.5</offset>
+            </li>
+          </capMods>
+      </li>
+      <li>
+        <minSeverity>0.4</minSeverity>
+        <label>lethal</label>

```

Comment:
What's the idea behind this stage?

Reply:
Knock the pawn out for roughly a day if the crash occurs. Something similar to food poisoning which happens in multiple stages and is gradual.